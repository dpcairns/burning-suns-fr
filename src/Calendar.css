.DatePicker {
    width: 350px;
    max-width: 100%;
    background: #d1c4e9;
    border: 1px solid green;
    font-family: Arial, Helvetica, sans-serif;
    line-height: 1.500em;
  }