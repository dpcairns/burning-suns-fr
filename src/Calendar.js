
import React, { Component } from 'react';
import Calendar from 'react-calendar';
import 'react-calendar/dist/Calendar.css';
import './Calendar.css'

export default class Sample extends Component {
    state = {
        value: new Date(),
    }

    onChange = value => this.setState({ value })

    render() {
        const { value } = this.state;
        console.log(value);

        return (
            <div className="Sample">
                <header>
                    <h1>Date of Journal</h1>
                </header>
                <div className="Sample__container">
                    <main className="Sample__container__content">
                        <Calendar
                            onChange={this.onChange}
                            hover
                            tileContent
                            value={value}
                        />
                    </main>
                </div>
            </div>
        );
    }

}