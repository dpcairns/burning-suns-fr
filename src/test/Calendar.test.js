import React from 'react';
import { shallow } from 'enzyme';
import Calendar  from '../Calendar'


describe('First React component test with Enzyme', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<Calendar />);
      expect(wrapper).toMatchSnapshot()
     });
 });
 