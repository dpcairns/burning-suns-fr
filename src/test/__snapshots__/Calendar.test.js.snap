// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme renders without crashing 1`] = `
<div
  className="Sample"
>
  <header>
    <h1>
      Date of Journal
    </h1>
  </header>
  <div
    className="Sample__container"
  >
    <main
      className="Sample__container__content"
    >
      <Calendar
        hover={true}
        maxDetail="month"
        minDetail="century"
        onChange={[Function]}
        returnValue="start"
        showNavigation={true}
        showNeighboringMonth={true}
        tileContent={true}
        value={2020-05-22T19:44:53.725Z}
      />
    </main>
  </div>
</div>
`;
